src/Components/test/E2ETest/Tests/BindTest.cs

```diff
@@ -640,5 +641,241 @@ public void CanBindTextboxGenericGuid()
             Browser.Equal(newValue, () => boundValue.Text);
             Assert.Equal(newValue, mirrorValue.GetAttribute("value"));
         }
+
+        // For date comparisons, we parse (non-formatted) values to compare them. Client-side and server-side
+        // Blazor have different formatting behaviour by default.
```

Comment:
What difference between client-side and server-side are you referring to? Nothing in the test code appears to vary its behavior or expectations based on the runtime type.

Reply:
This is because I'm parsing the text into `DateTime` objects and comparing those. The other tests for binding here compare strings directly. 

The differences are things like `01/01/0001` vs `1/1/0001`. ie: just annoying enough to make writing tests hard.